topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -1914,6 +1924,8 @@ For this, we will use NGINX. It is possible to configure Galaxy with Apache and
 >    ```
 >    {: data-commit="Configure nginx vhost for galaxy"}
 >
+>    You'll notice that we have set a 24 hour cache timeout on static content served by NGINX. [Google's PageSpeed Tools](https://developers.google.com/speed/pagespeed/insights/) can identify any additional compression or caching improvements you can make.
```

Comment:
y i k e s
![screenshot of pagespeed insights showing a 16 of 100 score for usegalaxy.org](https://user-images.githubusercontent.com/458683/122349336-7443da00-cf4c-11eb-82c4-ea187212dce1.png)

Reply:
.eu is little different

<img width="1005" alt="PageSpeed_Insights" src="https://user-images.githubusercontent.com/1814954/122351045-0bf5f800-cf4e-11eb-8afe-38a5b85dc58c.png">

Reply:
Definitely! It wasn't a dig at .org for sure, just galaxy's loadtime / huge amounts of JS in general.  (And EU is worse in some ways, displaying grafana really adds to the load time.)

Reply:
I am not starting a fight, I was just surprised there are substantial differences (e.g. time to interactive is 22s vs 30s)

Reply:
This is because of the different front page I assume. Try https://lite.usegalaxy.eu to measure more of Galaxy.

@hexylena I guess we could postpone loading the grafana iframe? And thinking about that are we lazy-loading the main iframe?

Reply:
Oh, same, sorry it came across as potentially combative. I'm not so surprised by the difference, I'm willing to be that's due to the toolbox size and then center panel content (grafana graphs are miserable, they use so many requests)

Reply:
![image](https://user-images.githubusercontent.com/458683/122353417-4fe9fc80-cf50-11eb-944a-4a154eb2fcb4.png)

1 point better :tada: hahah

it's a lot of unused JS, and the CSS caching could be better. But it's also just a LOT of time spent running JS. The joy of Vue/react/etc.

Reply:
the main page is relatively fast on its own

![image](https://user-images.githubusercontent.com/458683/122353893-c555cd00-cf50-11eb-9bc1-ef0b435b105d.png)